components/campaign-hero/manifest.json

```diff
@@ -0,0 +1,218 @@
+      "value": "green"
+    }
+  },
+  "functions": [
+    {
+      "name": "main",
+      "entry": "./dist/server.cjs",
+      "input": {
+        "type": "object",
+        "properties": {
+          "bkgConfig": {
+            "title": "Background",
+            "description": "Would you like to use an image or looping video as the background?",
+            "type": "object",
+            "required": ["type"],
+            "properties": {
+              "type": {
+                "title": "Background type",
+                "default": "Image",
+                "enum": ["Image", "Video"],
+                "type": "string"
+              }
+            },
+            "allOf": [
+              {
+                "if": {
+                  "required": [],
+                  "properties": {
+                    "type": {
+                      "const": "Image"
+                    }
+                  }
+                },
+                "then": {
+                  "required": ["bkgImage"],
+                  "properties": {
+                    "bkgImage": {
+                      "title": "Background image",
+                      "description": "Select a background image in landscape orientation. A 3x2 aspect ratio generally works well.",
+                      "type": "string",
+                      "format": "matrix-asset-uri"
+                    }
+                  }
+                }
+              },
+              {
+                "if": {
+                  "required": [],
+                  "properties": {
+                    "type": {
+                      "const": "Video"
+                    }
+                  }
+                },
+                "then": {
+                  "required": ["bkgVideo"],
+                  "properties": {
+                    "bkgVideo": {
+                      "title": "Background video",
+                      "description": "Please enter the full Vimeo URL, including https://.",
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            ]
+          },
+          "textConfig": {
+            "title": "Title & introduction",
+            "type": "object",
+            "required": ["title", "intro"],
+            "properties": {
+              "title": {
+                "title": "Heading",
+                "description": "The heading text",
+                "type": "string",
+                "default": "Lorem ipsum dolor"
+              },
+              "intro": {
+                "title": "Intro",
+                "description": "Introduction text that displays below the headline.",
+                "type": "FormattedText",
+                "default": "Lorem ipsum dolor"
+              }
+            }
+          },
+          "youtubeId": {
+            "title": "YouTube video ID",
+            "description": "The ID of the YouTube video that will play in a modal.",
+            "type": "string"
+          },
```

Comment:
As discussed with Justin, I've pulled the `youtubeId` field out of the quoteConfig group since it's more flexible for it to be independent of whether a quote is displayed.